.github/workflows/build.yml

```diff
@@ -559,6 +576,13 @@ jobs:
         with:
           name: PrismLauncher-${{ runner.os }}-${{ env.VERSION }}-${{ inputs.build_type }}-x86_64.AppImage
           path: PrismLauncher-${{ runner.os }}-${{ env.VERSION }}-${{ inputs.build_type }}-x86_64.AppImage
+      
+      - name: Upload AppImage Zsync (Linux)
+        if: runner.os == 'Linux' && matrix.qt_ver != 5
```

Comment:
Maybe also if release_type isn't debug?

Reply:
> Maybe also if release_type isn't debug?

I think this is fine. This would allow us to inspect zsync files without needing to create a release